bundles/AdminBundle/Controller/Admin/Asset/AssetController.php

```diff
@@ -833,7 +833,9 @@ protected function getThumbnailUrl(Asset $asset, array $params = [])
             return $this->generateUrl('pimcore_admin_asset_getdocumentthumbnail', $params);
         }
 
-        return null;
+        if ($asset instanceof Asset) {
+            return $asset->getFallbackThumbnailUrl();
```

Comment:
As this is entirely admin functionality, I'd just put the whole logic here. Imho the icon image shouldn't be defined in the model.